ui/app/pages/permissions-connect/permissions-connect.container.js

```diff
@@ -33,6 +40,16 @@ const mapStateToProps = (state, ownProps) => {
     addressLastConnectedMap[key] = formatDate(addressLastConnectedMap[key], 'yyyy-M-d')
   })
 
+  const connectPath = `${CONNECT_ROUTE}/${permissionsRequestId}`
+  const confirmPermissionPath = `${CONNECT_ROUTE}/${permissionsRequestId}${CONNECT_CONFIRM_PERMISSIONS_ROUTE}`
+
+  let page = null
+  if (pathname === connectPath) {
+    page = '1'
+  } else if (pathname === confirmPermissionPath) {
+    page = '2'
+  }
```

Comment:
Maybe worth throwing an error here in an `else` block? Or redirecting back to the home page?